spec/features/admin_can_log_in_to_dashboard_spec.rb

```diff
@@ -2,10 +2,10 @@
 RSpec.describe 'Admin', type: :feature do
   feature 'admin' do
     it 'can log in and view dashboard page' do
-      admin = User.create(username: 'admin', password: 'admin', role: 1)
+      admin = User.create(username: 'admin', password: 'admin', first_name: 'John', last_name: 'Smith', email_address: 'johnsmith@gmail.com', phone_number: '555-234-5678', role: 1)
```

Comment:
Line is too long. [180/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.